### DISCUSSION TITLE
我记得之前有个啥问题需要把缩放比默认改1.6来着

### DISCUSSION THREAD

**Discussion by Arondight** [Arondight/Adachi-BOT#641] [Q&A] [answered]
老大哥忘了改了吗 @Mark9804 今天看了眼日志突然想起来这事

**Comment by mark9804** [comment, marked as answer]
#603 那个啊，后来我想了想感觉这个解决方案不太对劲，如果真是不能正确处理小数问题的话，不管给到 1. 几，最后乘以一个奇数肯定还是只能得到含有小数点的内容

然后我从另一个角度考虑，因为我在 mac 上用来测试的这个仓库的 dev 分支的机器人设置 1.5 的时候传的图固定在最底下会有一条白边，但是生产环境设置成 2 的机器人就不会带这条白边，那如果调成 2 能消除掉的话就说明确实是半个像素引起的问题
<img width="500" alt="image" src="https://user-images.githubusercontent.com/9006264/152960765-af952f36-ba24-4c0f-94e9-6e6d2d853bf5.png">

然后我调成 2 ，mac 机器人发送的图片依然有这个问题，我就觉得算了，是玄学，不管了，就这样吧

不过米游社推送倒是一直都不正常，很多时候要重启机器人之后才会跟拉稀一样一股脑连着发十几条……这点上 dev bot 和生产 bot 倒是达成了共识

**Reply by Arondight** [reply]
我这里从来没有这个现象、感觉还是因为国外拉米游社图片不太好使

**Reply by mark9804** [reply]
我是从修复卡池数据被覆盖之后开始出现了这个问题，但是因为网络问题很玄学，也没找到问题出在哪，之后应该过两个月也换宽带了（去年九月初申请的服务，现在还 nm 跟我说要考虑怎么施工。。。。。。这就是工匠精神吗），就开摆了

要是方便的话老大哥给个新闻直接不抓取图片，只发送纯文字的选项？

**Reply by Arondight** [reply]
额，这个配置我不想加，我给加个函数的参数得了，自己改一下调用的代码
